entries/a/artists.spotify.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Spotify for Artists": {
+    "domain": "spotify.com",
+    "url": "artists.spotify.com",
```

```suggestion
    "domain": "artists.spotify.com",
```
It's probably better to have the domain as `artists.spotify.com` to avoid confusion with the existing `spotify.com` entry